app/views/dashboard/ng-partials/widgets/_build_sequence.html.haml

```diff
@@ -1,35 +1,35 @@
 .widget-wrapper
   .row
-    .small-12.columns
+    .col-sm-12
       .header-wrapper
         %h5 Build A Sequence
   .row
-    .small-12.columns
+    .col-sm-12
       .content-wrapper
         .row
-          .small-6.columns
+          .col-sm-6
             %input#right-label{"ng-model" => "sequence.name", :type => "text"}
-          .small-2.columns
+          .col-sm-2
             %button.green.button-like{"ng-click" => "saveSequence(sequence)"}
               Save
-          .small-2.columns
+          .col-sm-2
             %button.yellow.button-like{"ng-click" => "execute(sequence)"}
               Execute
-          .small-2.columns
+          .col-sm-2
             %button.red.button-like{"ng-click" => "deleteSequence(sequence)"}
               Delete
         .row
-          .small-12.columns
+          .col-sm-12
             .drag-drop-area.padding
               %h6 DRAG BASIC OPERATIONS AND/OR SAVE SEQUENCES HERE
         .row
-          .small-4.columns
+          .col-sm-4
             %label Sequence Parameters:
-          .small-5.columns
+          .col-sm-5
             %a.tiny.expand.button.round.dark-gray{:href => "#"} PLANT-ID/PLANT-GROUP-ID
-          .small-3.columns
+          .col-sm-3
             %a.tiny.expand.button.radius.dark-gray{:href => "#"} ADD PARAMETER
-          .small-12.columns{"as-sortable" => "dragControlListeners", "ng-model" => "sequence.steps"}
+          .col-sm-12{"as-sortable" => "dragControlListeners", "ng-model" => "sequence.steps"}
```

Comment:
Line is too long. [93/80]<br>Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace